common/server_player.cpp

```diff
@@ -1475,6 +1475,47 @@ Server_Player::cmdCreateToken(const Command_CreateToken &cmd, ResponseContainer
                                      attachedCard->getZone()->getPlayer()->getPlayerId());
             }
 
+            // Copy Arrows
+            const QList<Server_Player *> &players = game->getPlayers().values();
```

Comment:
Was this code block copied / modified from somewhere else? Would it make sense to functionize that if that's the case?